### PR TITLE
Added blurbs for callbacks, promises, and array-analysis

### PR DESCRIPTION
**Pull request by eakray**
This PR contains

- blurb for callbacks
- blurb for promises
- blurb for array-analysis

### CODE REVIEW

File: concepts/promises/.meta/config.json [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {
-  "blurb": "TODO: add blurb for promises concept",
+  "blurb": "A promise is an object that represents the eventual completion or failure of some asynchronous operation.",
```
Comment (SleeplessByte) [inline concepts/promises/.meta/config.json:2]: Nice

File: concepts/callbacks/.meta/config.json [hunk 2]
```diff
@@ -1,5 +1,5 @@
 {
-  "blurb": "TODO: add blurb for callbacks concept",
+  "blurb": "A callback is a functions passed as an argument to another function which invokes at the end of its execution",
```
Comment (SleeplessByte) [inline concepts/callbacks/.meta/config.json:2]: It's not necessarily at the end of the execution, and not necessarily invoked. Maybe a way to capture that too within the character limit?

File: concepts/array-analysis/.meta/config.json [hunk 3]
```diff
@@ -1,5 +1,5 @@
 {
-  "blurb": "TODO: add blurb for array-analysis concept",
+  "blurb": "Javascript provides a number of useful array methods for traversal, analysis and mutation.",
```
Comment (SleeplessByte) [inline concepts/array-analysis/.meta/config.json:2]: Nice. Perhaps traversal and mutation should be left out here, given the concept name.

**Review by tejasbubane — APPROVED** [review]
Thanks!

**Pull request merged**